locales/en/connection-tab.json

```diff
@@ -0,0 +1,29 @@
+{
+  "admin_url_button_aria_label": "More information about the URL for Kafka Admin REST API",
+  "admin_url_description": "Configure topics, consumer groups, and access for an individual Kafka instance.",
+  "admin_url_label": "URL for Kafka Admin REST API",
+  "admin_url_popover_body": "Use the REST API URL to connect to this Kafka instance over HTTPS to manage topics, consumer groups, and access.",
+  "admin_url_popover_label": "URL for Kafka Admin REST API",
+  "authentication_method": "Authentication method",
+  "bootstrap_server": "Bootstrap server",
+  "bootstrap_server_description": "Establish the initial connection between your client and Kafka instance.",
+  "create_service_account_to_generate_credentials": "Create a service account to generate credentials. Manage the service accounts you create on the <value>Service Accounts</value> page.",
+  "current_instance": "Accounts do not have access to this instance unless granted under the instance’s <value>Access tab</value>. The owner, the organization administrator, or users with the Allow Alter permission on this instance can manage access.",
+  "drawer_resource_tab_body_description_1": "Your application or tool initially connects to the Kafka instance using the bootstrap server and authenticates with your service account credentials. You can also access the Kafka instance by calling the REST API using a token obtained from Red Hat SSO.",
+  "kafka_service_fleet_manager_button_aria_label": "More information about the URL for Kafka Service Fleet Manager",
+  "kafka_service_fleet_manager_description": "Manage Kafka instances and service accounts, and access metrics.",
+  "kafka_service_fleet_manager_popover_body": "Use the REST API URL to connect to manage Kafka instances and service accounts, and to access metrics.",
```

Comment:
@wtrocki @jbyrne-redhat  - The other popover includes the following contents:
> Use the REST API URL to connect to this Kafka instance over HTTPS to manage topics, consumer groups, and access.

Should we also include the "over HTTPS" part here too? Or is this one special?

Reply:
I think that can be skipped.
Rest implies using HTTP protocol (of which https is just special case)

Reply:
Thanks, @wtrocki! Just to confirm, you're suggesting we can remove the text "over HTTPS" from the following popover?

![image](https://user-images.githubusercontent.com/21063328/175963548-8ff50b2f-681e-480d-a9cd-b58dbd01389b.png)